### PR TITLE
Draft

### PR DESCRIPTION
**Pull request by dchess**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -153,11 +153,85 @@ def load_participants(self):
                 page_token = response.get("next_page_token")
                 params["next_page_token"] = page_token
 
+    def load_groups(self):
```
Comment (susanxiong) [inline main.py:156]: add a docstring

File: main.py [hunk 2]
```diff
@@ -153,11 +153,85 @@ def load_participants(self):
...
+                    members["groupId"] = group_id
+                    self.sql.insert_into(table_name, members)
+                    logging.info(f"Inserted {len(members)} records into {table_name}")
+                    page += 1
+                    params["page"] = page
+
+    def _get_students(self):
+        """Query database for new students without Zoom accounts"""
+        return pd.read_sql_table(
+            "vw_Zoom_NewStudentAccounts", con=self.sql.engine, schema=self.sql.schema
```
Comment (susanxiong) [inline main.py:200]: Should this view definition be added to the repo?

File: main.py [hunk 3]
```diff
@@ -153,11 +153,85 @@ def load_participants(self):
...
+        )
+
+    def create_student_accounts(self):
+        """Create Zoom accounts for students and add to Students group"""
+        students = self._get_students()
+        students["type"] = 1
```
Comment (susanxiong) [inline main.py:206]: What does the 1 mean here?

File: main.py (hunk 2 above)
Comment (dchess) [reply main.py:201]: It will be in sz_custom

**Pull request merged**